enterprise-suite/templates/prometheus.yaml

```diff
@@ -198,9 +197,21 @@ metadata:
   labels:
     app: prometheus
     component: server
-  name: prometheus-server
+  name: bare-prometheus
 data:
-{{ (tpl (.Files.Glob "prometheus-server/*").AsConfig .) | indent 2 }}
+  prometheus.yml: |-
+    # this is a placeholder to keep prometheus running while it waits the real config
+    global:
+      scrape_interval:     1m
+      evaluation_interval: 1m
+    scrape_configs:
+      - job_name: wait
+        static_configs:
+          - targets: ['localhost:9090']
+        metric_relabel_configs:
+        - source_labels: [__name__]
+          regex: .+
+          action: drop
```

Comment:
Any reason to not put this in `enterprise-suite/es-monitor-api`? Then it can just be included like the other files in there. Sort of nicer I think than inline yaml.

Reply:
Yeah this is for the init container to set things in place before es-monitor-api and prometheus are started.  The idea is for prometheus to be able to start with a bare config, and essentially wait for the real config instead of exiting.